### PR TITLE
Fix/tpi/home page hero styles

### PR DESCRIPTION
**Pull request by rapzo**

### CODE REVIEW

**Comment by aabdaab** [comment]
Hey, looking good 👍
Couple of things we need to fix here though. 
1. The hero pictures here are of too low quality; not sure if these are some old ones? We've got a higher quality ones in the assets on Invision (I downloaded them and compared with the ones in repo and they're different)
image
☝️ these ones are pixelized
2. Also, we've got a `homepage pattern` in assets that will fix the `cover` issue that we have now.
image
3. Background-color should be different: `#2465F5` on designs
4. The dots pattern for Investors & Supporters should be different too, we probably have a correct one in the assets
5. Mobile secondary hero; I think we need to remove the padding from the section title on mobile as the title spans through two lines
image
6. Mobile secondary hero - could you double-check the spacings? It seems that they are too big, the whole section should fit in the mobile screen.

File: app/assets/stylesheets/tpi.scss [hunk 1]
```diff
@@ -40,3 +40,9 @@
 html {
   scroll-behavior: smooth;
 }
+
+body {
+  @include mobile {
```
[the remark this answers is not in the document]
Comment (rapzo) [reply app/assets/stylesheets/tpi.scss]: Will do 👍

**Comment by rapzo** [comment]
1: I'm using the lowest quality one there, yeah, maybe the 2x should be better as a default, although any 
2: that's the pattern right there, but it doesn't play well due to fixed sizes in the svg itself
4: there's no asset for that, the svg logo was the closest i could find

Both scenarios require svg handling. I can go for it but I can't predict the outcome nor the time it will take.

Everything else, damn! That's some sharp eye right there!

**Comment by simaob** [comment]
Hey @batraisk do you mind picking this PR up, update it with the base branch and address Marta's comments. You might need to ask @faustoperez for some assets that Rui said were missing. If it requires svg handling let's not do it for now. Just the simplest changes! Thanks!

**Comment by batraisk** [comment]
@simaob , Of course, I’ll do it now)

**Comment by batraisk** [comment]
@simaob , I've finished. check please)

**Comment by simaob** [comment]
I think we got some issue with the button's colors, they should be blue they are not!

Screenshot 2020-03-13 at 12 02 23

**Comment by batraisk** [comment]
fixed

**Pull request merged**